core.js

```diff
@@ -488,8 +514,17 @@ Velocity = Velocity || {};
     VelocityTestFiles.remove({});
     VelocityFixtureFiles.remove({});
 
-    _watcher = chokidar.watch(Velocity.getTestsPath(), {ignored: /[\/\\]\./, persistent: true});
+    var paths = [Velocity.getTestsPath()];
+
+    _.each(fs.readdirSync(Velocity.getPackagesPath()), function(dir) {
+      if (fs.lstatSync(Velocity.getPackagePath(dir)).isDirectory() && fs.existsSync(Velocity.getTestsPath(dir))) {
```

Comment:
Isn't the existsSync check unnecessary because isDirectory implicates this? We should catch exceptions from lstatSync.